cmd/cmd.go

```diff
@@ -442,7 +443,7 @@ func resolveExitCode(err error) {
 		os.Exit(exitCodeFileNotFound)
 	case unwrapped == errorUncategorized:
 		os.Exit(exitCodeUncategorizedError)
-	case unwrapped == accounting.ErrorMaxTransferLimitReached:
+	case stderrors.Is(err, accounting.ErrorMaxTransferLimitReached):
```

Comment:
We can't use `errors.Is` yet as we still need to support go1.11

However you shouldn't need it as we use `fserrors.Cause` which does a pretty similar thing.